src/lib/TypeWriter.svelte

```diff
@@ -52,12 +56,13 @@
 	}
 
 	const blink = async (iterations: number) => {
```

Comment:
It was an oversight on my part when I first made the change, but this function can, and should be synchronous
```suggestion
	const blink = (iterations: number) => {
```